rotkehlchen/db/upgrades/v38_v39.py

```diff
@@ -251,15 +251,36 @@ def _update_evm_transaction_data(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _update_evm_transaction_data')
 
 
-def _add_arbitrum_one_location_and_nodes(write_cursor: 'DBCursor') -> None:
-    log.debug('Enter _add_arbitrum_one_location_and_nodes')
+def _add_arbitrum_one_location(write_cursor: 'DBCursor') -> None:
+    log.debug('Enter _add_arbitrum_one_location')
     write_cursor.execute('INSERT OR IGNORE INTO location(location, seq) VALUES ("i", 41);')
+    log.debug('Exit _add_arbitrum_one_location')
+
+
+def _update_rpc_nodes_table(write_cursor: 'DBCursor') -> None:
+    log.debug('Enter _update_rpc_nodes_table')
+    write_cursor.execute('SELECT name, endpoint, owned, active, weight, blockchain FROM rpc_nodes')
```

Comment:
Can you add the existence check here? The same pattern we use in upgrades when we want to act differently when something exists or not?